src/app/clients/clients-view/client-actions/add-client-charge/add-client-charge.component.ts

```diff
@@ -124,7 +127,7 @@ export class AddClientChargeComponent implements OnInit {
           clientCharge.feeOnMonthDay = this.datePipe.transform(prevDate, monthDayFormat);
```

Comment:
@karantakalkar the date being sent over here is in a different compared to others, I couldn't make a request to this endpoint because this was under an if block and couldn't be tested. Should this remain in this format only, or changed to the dd mmmm yyyy format.
P.S: Every endpoint data changed has been checked thoroughly. I couldn't access this endpoint, so didn't update this field.

Reply:
No dont change it, leave this hardcoded